utilities/decrypt_data/decrypt_object.py

```
@@ -0,0 +1,144 @@
+        result = resp.json()           # e.g. [ {key1: plain1}, {key2: plain2} ]
+        logger.debug("Decrypt API returned %d result entries", len(result))
+        
+        out = {}
+        for entry in result:
+            out.update(entry)
+        
+        logger.info("Successfully decrypted %d fields: %s", len(out), list(out.keys()))
+        return out
+
+    except requests.exceptions.Timeout as e:
+        logger.error("Decrypt API timeout after 10s for fields %s: %s", list(payload.keys()), e)
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)